.landscaper/container-deployer/example/imports.yaml

```diff
@@ -0,0 +1,34 @@
+imports:
+  cluster:
+    apiVersion: landscaper.gardener.cloud/v1alpha1
+    kind: Target
+    metadata:
+      name: cluster
+      namespace: default
+    spec:
+      type: landscaper.gardener.cloud/kubernetes-cluster
+      config:
+        kubeconfig: |
+          apiVersion:...
+  releaseNamespace: container-deployer
+  releaseName: landscaper-container-deployer
+  values: |
+    deployer:
+      namespace: ""
+      oci: 
+        allowPlainHttp: false
+        secrets: {}
+
+    targetSelector:
+      - annotations:
+        - key:
+          operator:
+          value:
+
+    replicaCount: 1
+
+    image:
+      repository: eu.gcr.io/gardener-project/landscaper/container-deployer-controller
```

Comment:
should not be set by any input data as it is read from the component descriptor

Reply:
Fixed.